src/current/_includes/common/network-partitions.md

```diff
@@ -0,0 +1,7 @@
+A network partition occurs when two or more nodes are prevented from communicating with each other in one or both directions. A network partition can be caused by a network outage or a configuration problem with the network, such as when allowlisted IP addresses or hostnames change after a node is torn down and rebuilt.
+
+In a **symmetric** partition, node communication is disrupted in both directions. In an **asymmetric** partition, nodes can communicate in one direction but not the other.
+
+CockroachDB protects against asymmetric partitions by converting all asymmetric (uni-directional) network partitions into symmetric (bi-directional) network partitions. This increases cluster resiliency by reducing the number of partition-related failures that can occur.  Many temporary symmetric partitions can be recovered from automatically without operator intervention.
+
+The effect of a network partition depends on which nodes are partitioned, where the ranges are located, and to a large extent, whether [localities]({% link {{ page.version.version }}/cockroach-start.md %}#locality) or [zone configurations]({% link {{ page.version.version }}/configure-replication-zones.md %}) are defined. A partition that cuts off at least `(n-1)/2` replicas from a [range]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range) will cause range unavailability which will cause some data unavailability. If there are no localities or other constraints on where replicas are placed, then a partition of any ([`num_replicas`]({% link {{ page.version.version }}/configure-replication-zones.md %}#num_replicas) / 2) nodes will likely cause unavailability.
```

Comment:
```suggestion
The effect of a network partition depends on which nodes are partitioned, where the ranges are located, and to a large extent, whether [localities]({% link {{ page.version.version }}/cockroach-start.md %}#locality) or [zone configurations]({% link {{ page.version.version }}/configure-replication-zones.md %}) are defined. A partition that cuts off at least `(n-1)/2` replicas from a [range]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range) will cause range unavailability, which will cause some data unavailability. If there are no localities or other constraints on where replicas are placed, then a partition of any (`[`num_replicas`]({% link {{ page.version.version }}/configure-replication-zones.md %}#num_replicas)/2`) nodes will likely cause unavailability.
```

Reply:
sorry can't do this one, it breaks the link rendering 

![Screenshot 2024-10-10 at 2 20 19 PM](https://github.com/user-attachments/assets/3bde9704-6d19-4861-b81f-ece16c6e3d09)